packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -429,6 +474,9 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     switch (newState) {
       case ExecutionEngineState.ONLINE:
         this.logger.info("Execution client became online", {oldState, newState});
+        this.getClientVersion().catch((e) => {
```

Comment:
The online state is good place to trigger this as the online time this will be the `newState` if el was offline but there is a small issue on startup with this as initial state is online, see https://github.com/ChainSafe/lodestar/blob/8cb08ff56f66022c84d8ca1a91e82dba290e644a/packages/beacon-node/src/execution/engine/http.ts#L106

I would assume it's currently not fetched on startup, other than handling inital fetch this looks good to me